### PR TITLE
Websites: Clarify deploy and tag stage

### PR DESCRIPTION
**Pull request by agcolom**
Fixes gh-116

### CODE REVIEW

File: pages/web-sites.md [hunk 1]
```diff
@@ -120,8 +120,14 @@ For other sites:
 
 To create a tag, use `npm version [major | minor | patch]`.
 
-Afterwards, make sure to push both version change commit and the tag:
-`git push --tags origin master`
+Afterwards, make sure to push both version change commit and the tag to the main jQuery repo.
+
+<div class="warning">You'll want to check your remote setup using ```git remote -v``` and check which remote is used for the main jQuery repo.</div>
```
Comment (arthurvr) [inline pages/web-sites.md]: This doesn't render correctly. Check the GitHub preview of this markdown file. From the [markdown spec](http://daringfireball.net/projects/markdown/syntax#html):

> Note that Markdown formatting syntax is not processed within block-level HTML tags. E.g., you can’t use Markdown-style *emphasis* inside an HTML block.

Comment (agcolom) [inline pages/web-sites.md]: @arthurvr It all looks fine for me when deployed locally. Should I use single ``` ` ``` instead?

Comment (arthurvr) [inline pages/web-sites.md]: well, this is how it looks on GitHub:
contribute_jquery_org_web-sites_md_at_clarifydeploy_ _agcolom_contribute_jquery_org

If it works fine for our sites that's in fact a buggy behaviour of our markdown parser so I'd prefer to fix it (just use html `<code>` syntax directly)

-

Oh and yeah, except for this line, let's be consistent and use a single backtick for inline code instead.

**Comment by agcolom** [comment]
@arthurvr Yes, I think i've fixed it here

**Comment by mzgol** [comment]
Looks fine but I have one doubt - are there more places where we describe steps for maintainers that might have the repo set up to have an `origin` upstream remote? This would be quite repetitive to put this same text in all those places.

File: pages/web-sites.md [hunk 2]
```diff
@@ -120,8 +120,14 @@ For other sites:
 
 To create a tag, use `npm version [major | minor | patch]`.
 
-Afterwards, make sure to push both version change commit and the tag:
-`git push --tags origin master`
+Afterwards, make sure to push both version change commit and the tag to the main jQuery repo.
```
Comment (scottgonzalez) [inline pages/web-sites.md]: I don't think this change is necessary, especially with the warning included.

File: pages/web-sites.md [hunk 3]
```diff
@@ -120,8 +120,14 @@ For other sites:
...
+
+<div class="warning">You'll want to check your remote setup using <code>git remote -v</code> and check which remote is used for the main jQuery repo.</div>
```
Comment (scottgonzalez) [inline pages/web-sites.md]: Perhaps:

The example below uses `upstream` for the jQuery organization on GitHub. Use <code>git remote -v</code> to ensure you're pushing to the correct remote repo.

Comment (scottgonzalez) [inline pages/web-sites.md]: And then drop the condition below.

File: pages/web-sites.md [hunk 4]
```diff
@@ -120,8 +120,14 @@ For other sites:
...
+
+If `upstream` is the remote you use for that, type:
+`git push --tags upstream master`
+
+If you use `origin` for the main jQuery repo, replace `upstream` with `origin` in the command above.
```
Comment (scottgonzalez) [inline pages/web-sites.md]: I personally don't like the words "main jQuery repo."

Comment (scottgonzalez) [inline pages/web-sites.md]: I probably should've read the whole thing before I started commenting. Perhaps take my suggested wording and move it down here, replacing "below" and "above" and adding this sentence. That would give you:

> The example above uses `upstream` for the repo in the jQuery organization on GitHub. Use `git remote -v` to ensure that you're pushing to the correct remote repo. For example, If you use `origin` for the repo in the jQuery organization, replace `upstream` with `origin`.

**Comment by agcolom** [comment]
@scottgonzalez Thanks for the feedback. I've updated the PR with your comments, and put it in a warning div. 
@mzgol I may have missed it but I have never come across such an explanation anywhere on our sites.

**Comment by mzgol** [comment]
> @mzgol I may have missed it but I have never come across such an explanation anywhere on our sites.

I meant that this notice will make only maintainers that go to this specific site see the notice about `upstream`/`origin`, others won't. So for it to be complete it should either be everywhere or nowhere; but "everywhere" will mean either repetition or linking to a common source.

**Comment by agcolom** [comment]
@mzgol oh I see. I suppose we could in that case copy the same warning div to this page: http://contribute.jquery.org/repo-maintainers-guide/ where we first mention pushing to upstream. @scottgonzalez What do you think?

**Comment by agcolom** [comment]
Or we put the warning just before the heading http://local.contribute.jquery.org/repo-maintainers-guide/#fixing-commits (so just after we mention both `origin` and `upstream`) and say

The examples above uses `upstream` for the repo in the jQuery organization on GitHub. Use `git remote -v` to ensure that you're pushing to the correct remote repo. For example, if you use `orgin` for the repo in the jQuery organization, and `ustream` for your fork, swap `upstream` with `origin`.

**Comment by mzgol** [comment]
> For example, if you use (..) ustream for your fork.

This seems weird, I doubt many people will name their forks `upstream`; it's more likely they'll use their own name or sth else. I'd remove that part.

That page already has this:
> Assuming you have the `jquery` repo set as your `upstream` remote

so maybe here it's already covered. Are there more sites where this would be an issue?

**Comment by agcolom** [comment]
@mzgol indeed, that's at the top of the maintainer's guide page. So I'd agree with you, we don't need to add more on that page. I think all other pages are covered. So I think for now we only need to fix the page I'm taking care of in this PR.

**Comment by mzgol** [comment]
> I think all other pages are covered. So I think for now we only need to fix the page I'm taking care of in this PR.

LGTM then!

**Comment by scottgonzalez** [comment]
> @scottgonzalez What do you think?

I think you should do whatever makes sense for the people who need the documentation :-)

**Pull request closed without merging**